torch/_inductor/subgraph_lowering.py

```diff
@@ -87,7 +88,7 @@ def placeholder(self, idx):
 
     def output(self, *args):
         return self.tracer.create_node(
-            "output", "output", tuple(self.tracer.create_arg(a) for a in args), {}
+            "output", "output", (tuple(self.tracer.create_arg(a) for a in args),), {}
```

Comment:
why the wrapping and unwrapping in line 66?

Reply:
Apparently the fx output operator doesn't actually support multiple arguments, it will silently ignore anything but `arg[0]`. So you need to explicitly pass a tuple of results.